### PR TITLE
Remove manual depth projection from car and nyud examples

### PR DESCRIPTION
**Pull request by emilk**
The depth point clouds are now much better in every way, and _much_ faster.

Let's simplify and speed up the examples!

In fact, we can now log all depth images in `nyud`, because the costly depth projection is gone.

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: examples/python/notebook/cube.ipynb [hunk 1]
```diff
@@ -177,7 +177,6 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "\n",
```
Comment (teh-cmc) [inline examples/python/notebook/cube.ipynb:180]: Wasn't this there on purpose?

**Review by teh-cmc — APPROVED** [review]

File: examples/python/notebook/cube.ipynb (hunk 1 above)
Comment (emilk) [reply examples/python/notebook/cube.ipynb:180]: I have no idea what happened here - good catch

File: examples/python/notebook/cube.ipynb [hunk 2]
```diff
@@ -177,7 +177,6 @@
    "metadata": {},
    "outputs": [],
    "source": [
```
Comment (emilk) [inline examples/python/notebook/cube.ipynb:179]:
```suggestion
   "source": [
       "\n",
```

File: examples/python/notebook/cube.ipynb [hunk 3]
```diff
@@ -177,7 +177,7 @@
...
-    "\n",
+       "\n",
```
Comment (emilk) [inline examples/python/notebook/cube.ipynb:180]:
```suggestion
     "\n",
```

File: examples/python/notebook/cube.ipynb [hunk 4]
```diff
@@ -177,7 +177,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "\n",
+     "\n",
```
Comment (emilk) [inline examples/python/notebook/cube.ipynb:180]:
```suggestion
    "\n",
```

**Pull request merged**